fe1-web/src/core/components/BackButton.tsx

```diff
@@ -1,23 +1,45 @@
 import { useNavigation } from '@react-navigation/core';
 import { StackNavigationProp } from '@react-navigation/stack';
+import PropTypes from 'prop-types';
 import React from 'react';
 
 import { AppParamList } from 'core/navigation/typing/AppParamList';
 import { Color, Icon } from 'core/styles';
 
+import NavigationPadding from './NavigationPadding';
 import PoPIcon from './PoPIcon';
 import PoPTouchableOpacity from './PoPTouchableOpacity';
 
 type NavigationProps = StackNavigationProp<AppParamList>;
 
-const BackButton = () => {
+const BackButton = ({ padding }: IPropTypes) => {
   const navigation = useNavigation<NavigationProps>();
 
   return (
-    <PoPTouchableOpacity onPress={navigation.goBack}>
-      <PoPIcon name="arrowBack" color={Color.inactive} size={Icon.size} />
-    </PoPTouchableOpacity>
+    <>
+      <PoPTouchableOpacity onPress={navigation.goBack}>
+        <PoPIcon name="arrowBack" color={Color.inactive} size={Icon.size} />
+      </PoPTouchableOpacity>
+      <NavigationPadding n={padding || 0} />
+    </>
   );
 };
 
+const propTypes = {
+  padding: PropTypes.number,
+};
+
+BackButton.propTypes = propTypes;
+
+BackButton.defaultProps = {
+  padding: 0,
+};
+
+type IPropTypes = PropTypes.InferProps<typeof propTypes>;
+
 export default BackButton;
+
+/**
+ * Returns a back button with the padding parameter set accordingly
+ */
+export const makeBackButton = (padding: number) => () => <BackButton padding={padding} />;
```

Comment:
I now think you're right. I though we might need it like the `makeIcon` function (https://github.com/dedis/popstellar/blob/a71f859b97c8197fa24910951ead64d87a6b4d95/fe1-web/src/features/notification/navigation/NotificationNavigation.tsx#L43). Directly putting `() => <BackButton padding={padding} />` inside the navigator component is not good because it will create a new react component on each render but if we do that beforehand, i.e. when using the screen props like we usually do, it is indeed not a problem! :)